common/src/main/java/com/wynntils/models/abilities/type/ShieldType.java

```diff
@@ -32,5 +32,9 @@ public boolean verifyShield(ArmorStand armorStand) {
         return verifyArmorStand(armorStand);
     }
 
+    protected boolean clearOnSpell() {
```

Comment:
```suggestion
    protected boolean shouldClearOnSpellCast() {
```

Perhaps a more telling name. I think a comment regarding the method would help us in the future. "Whether the shield should be cleared when a spell is cast".